webpack/components/extensions/HostDetails/Tabs/ModuleStreamsTab/ModuleStreamsTab.js

```diff
@@ -277,45 +363,223 @@ export const ModuleStreamsTab = () => {
+                    }}
+                    isDisabled={!upgradable}
+                  >
+                    {__('Update')}
+                  </DropdownItem>,
+                  <DropdownItem
+                    key={`dropdownItem-reset-${id}`}
+                    component="button"
+                    onClick={() => {
+                      setActionableModuleSpec(moduleSpec);
+                      setModuleStreamAction('reset');
+                      setActionModalOpen(true);
+                      setUseCustomizedRex('');
+                      setDropdownOpen('');
+                    }}
+                  >
+                    {__('Reset')}
+                    <InactiveText style={{ marginBottom: '1px' }} text={__('Reset to the default state')} />
+                  </DropdownItem>,
+                  <DropdownItem
+                    key={`dropdownItem-remove-${id}`}
+                    component="button"
+                    onClick={() => {
+                      setActionableModuleSpec(moduleSpec);
+                      setModuleStreamAction('remove');
+                      setActionModalOpen(true);
+                      setUseCustomizedRex('');
+                      setDropdownOpen('');
+                    }}
+                  >
+                    {__('Remove')}
+                    <InactiveText style={{ marginBottom: '1px' }} text={__('Uninstall and reset')} />
+                  </DropdownItem>,
+                );
+              }
+              return (
+                <Tr key={`${id} ${index}`}>
+                  <Td>
+                    <a
+                      href={`/module_streams?search=module_spec%3D${moduleSpec}+and+host%3D${hostName}`}
+                    >
+                      {name}
+                    </a>
+                  </Td>
+                  <Td>
+                    <StreamState moduleStreamStatus={moduleStreamStatus} />
+                  </Td>
+                  <Td>{stream}</Td>
+                  <Td>
+                    <EnabledIcon
+                      streamText={moduleStreamStatus}
+                      streamInstallStatus={installedProfiles}
+                      upgradable={upgradable}
+                    />
+                  </Td>
+                  <Td>
+                    <HostInstalledProfiles
+                      moduleStreamStatus={moduleStreamStatus}
+                      installedProfiles={installedProfiles}
+                    />
+                  </Td>
+                  <Td key={`actions-td-${id}-${dropdownOpen}`}>
+                    <Dropdown
+                      key={`actions-dropdown-${id}-${dropdownOpen}`}
+                      isPlain
+                      style={{ width: 'inherit' }}
+                      position={DropdownPosition.right}
+                      toggle={
+                        <KebabToggle onToggle={() => (dropdownOpen === id) ? setDropdownOpen('') : setDropdownOpen(id)} id={`toggle-dropdown-${id}`} />
```

Comment:
```suggestion
                        <KebabToggle onToggle={() => ((dropdownOpen === id) ? setDropdownOpen('') : setDropdownOpen(id))} id={`toggle-dropdown-${id}`} />
```
I think this will placate eslint..